consoles/sshVirtsh.pm

```diff
@@ -18,11 +18,11 @@ package consoles::sshVirtsh;
 use base 'consoles::sshXtermVt';
```

Comment:
we really should do something about this as well. Because in here we use base 'SINGLE' and in other cases use base qw(SINGLE).

base 'SINGLE' sounds like the more reasonable thing to do - perhaps there is a perlcritic for that